tests/DMSDocumentTest.php

```diff
@@ -121,6 +134,17 @@ public function testGetActionTaskHtml()
         $this->assertContains('<li class="ss-ui-button dmsdocument-action" data-panel="', $result);
         $this->assertContains('permission', $result);
         $this->assertContains('Example', $result);
+
+        $actions = array('example', 'embargo','find-usage');
+        foreach ($actions as $action) {
+            // Test remove with string
+            $document->removeActionPanelTask($action);
+        }
+        $result = $document->getActionTaskHtml();
+
+        $this->assertNotContains('Example', $result);
+        $this->assertNotContains('embargo', $result);
+        $this->assertNotContains('find-usage', $result);
```

Comment:
Could you add a positive assertion here? Otherwise `removeActionPanelTask` could remove all of them and this would still pass